util/envDynawoAlgorithms.sh

```diff
@@ -196,7 +196,7 @@ set_environnement() {
   export_var_env DYNAWO_ALGORITHMS_HOME=UNDEFINED
   export_git_branch
   export_var_env_force DYNAWO_ALGORITHMS_SRC_DIR=$DYNAWO_ALGORITHMS_HOME
-  export_var_env DYNAWO_ALGORITHMS_DEPLOY_DIR=$DYNAWO_ALGORITHMS_HOME/deploy/$DYNAWO_COMPILER_NAME$DYNAWO_COMPILER_VERSION
+  export_var_env DYNAWO_ALGORITHMS_DEPLOY_DIR=$DYNAWO_ALGORITHMS_HOME/deploy/$DYNAWO_COMPILER_NAME$DYNAWO_COMPILER_VERSION/dynawo-algorithms
```

Comment:
so in the zip can you verify we add a dynawo-algorithms top folder instead of having bin, lib... at the root, issue with taht is unzip name.zip creates lib, bin dirs right there where you extract and it's annoying